projects/example-app/src/app/auth/actions/auth-api.actions.ts

```diff
@@ -0,0 +1,47 @@
+import { Action } from '@ngrx/store';
+import { User, Credentials } from '../models/user';
+
+export enum AuthApiActionTypes {
```

Comment:
The `Logout`, `LogoutConfirmation` and `LogoutDismiss` actions should still be in the `auth.actions.ts` file with just `[Auth]` as the category.